pkg/reconciler/openshift/tektonaddon/pipelinesascode.go

```diff
@@ -86,3 +94,12 @@ func (r *Reconciler) ensurePAC(ctx context.Context, ta *v1alpha1.TektonAddon) er
 
 	return nil
 }
+
+// getPACImage gets the PAC image name from the IMAGE_PAC_PIPELINES_AS_CODE env var
+func getPACImage() (string, error) {
+	pacImage := os.Getenv(common.PACImageEnv)
+	if pacImage == "" {
+		return "", fmt.Errorf("environment variable %v not found", common.PACImageEnv)
```

Comment:
there can be a case like this env is not present in the development cycle, or in k8s target for now

Reply:
fixed in https://github.com/tektoncd/operator/pull/664/commits/1d5be1021ef4faedc26493b48628efb90e6012b8